### ISSUE TITLE
[Debian] package_versions table filled with more entries with the same package version.

### ISSUE THREAD

**Issue by michelescarlato** [fasten-project/fasten#442]
@gdrosos @MagielBruntink 
Dear Giorgos and Magiel,

I was troubleshooting the feeder locally, after deploying it on Monster, and I am observing a strange feeding approach for the package_versions table.

Here goes the table output :
 
```
root@b925ba175cc8:/# PGPASSWORD=fasten1234 psql --host localhost --port 5432 --username fasten fasten_c --command="SELECT * FROM package_versions;"
 id | package_id |  version  | cg_generator | artifact_repository_id | architecture |     created_at      |                         metadata      
                    
----+------------+-----------+--------------+------------------------+--------------+---------------------+---------------------------------------
--------------------
 -1 |         -1 | 0.0.1     | CScout       |                        |              |                     | 
  1 |          1 | 1.13.10-1 | cscout       |                        | amd64        | 2019-01-06 18:52:02 | {"licenses": [{"name": "", "source": "
DEBIAN_PACKAGES"}]}
  2 |          2 | 5.2.8-1   | cscout       |                        | amd64        | 2018-12-27 21:01:17 | {}
  3 |          3 | 5.2.8-1   | cscout       |                        | amd64        | 2018-12-27 21:01:17 | {}
  4 |          4 | 5.2.8-1   | cscout       |                        | amd64        | 2018-12-27 21:01:17 | {}
  5 |          5 | 5.2.8-1   | cscout       |                        | amd64        | 2018-12-27 21:01:17 | {}
  6 |          6 | 5.2.8-1   | cscout       |                        | amd64        | 2018-12-27 21:01:17 | {}
(7 rows)

```

Is any of you aware of this?
Why there are these entries with the same package version in this table?

This behavior occurs with several packages, particularly I observed it for:
- **libgda5** _version_ 5.2.8-1
- **avrdude** _version_ 6.3-20171130+svn1429-2 

I am not sure this is a bug, but I labeled it in this way.

If you want to reproduce this behavior feel free to use the [debian-license-detector](https://github.com/fasten-project/fasten-docker-deployment/tree/debian-license-detector) DC branch, using [this](https://github.com/fasten-project/fasten-docker-deployment/blob/debian-license-detector/test-resources/debian.coords.txt) file to feed the pipeline (which includes lbgda5).

**Comment by MagielBruntink** [comment]
Does the duplication behavior also occur without the Debian license feeder? I think the metadata plugin should not insert duplicate package_versions, but that should be validated first.

**Comment by MagielBruntink** [comment]
Hi @michelescarlato,

I did a local run with the coordinates mentioned by you. What I see happening is that cscout generates several output packages the libgda coordinate, with have same version but different package_names. I think this is normal, but @gdrosos and team should verify. Below a screenshot of the DB content after the run:

<img width="198" alt="image" src="https://user-images.githubusercontent.com/3635696/163857033-d2cbb8af-788a-44db-9f23-5880b197e472.png">

**Comment by michelescarlato** [comment]
Ok, thanks for the clarification.
I wait to see the fix in [#6](https://github.com/fasten-project/debian-builder/issues/6) because running the pipeline with this Cscout version is causing problems to the license detector and the feeder.

Meanwhile I close this issue.
